app/templates/home.html

```diff
@@ -6,11 +6,16 @@ <h1 class="sidebar-title">Bloc Chat</h1>
     </div>
     <div class="room-list">
       <div ng-repeat="room in home.rooms">
-        <h3 class="sidebar-room">{{ room.$value }}</h3>
+        <h3 class="sidebar-room" ng-click="home.changeRoom(room)">{{ room.$value }}</h3>
       </div>
     </div>
   </section>
   <section class="chat-room">
-    <p>This is where the chat room content will go.</p>
+    <h1 class="active-room-title">{{ activeRoom.$value }}</h1>
```

Comment:
Refer to this as `home.activeRoom` in order to access the variable on the controller